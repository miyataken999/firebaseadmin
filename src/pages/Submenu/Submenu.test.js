import React from 'react';

import Submenu from '.';

describe('<Submenu /> rendering', () => {
  it('should render without crashing', () => {
    const { component } = renderWithProviders(<Submenu />)({});

    expect(component.asFragment()).toMatchSnapshot();
  });
});
