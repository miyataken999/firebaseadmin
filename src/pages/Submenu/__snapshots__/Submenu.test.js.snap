// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Submenu /> rendering should render without crashing 1`] = `
<DocumentFragment>
  <section
    class="hero is-hero-bar"
  >
    <div
      class="hero-body"
    >
      <h1
        class="title"
      >
        Submenu
      </h1>
    </div>
  </section>
  <section
    class="section is-main-section"
  >
    Submenu content
  </section>
</DocumentFragment>
`;
